Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -384,7 +384,8 @@ function Invoke-CodeGen {
     param(
         [ValidateSet('SQLServer', 'PostgreSQL')]
         [String] $Engine,
-        [switch] $IncludePlugins
+        [switch] $IncludePlugins,
+        [String] $ExtensionPaths
```

Comment:
should this be a string array? then you don't have to do the splitting below
```suggestion
        [string[]] $ExtensionPaths
```

Reply:
I changed it . Thanks